### PR TITLE
trees done, one test is messed up

### PR DESCRIPTION
**Pull request by ryanapodaca**
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

### CODE REVIEW

**Comment by ryanapodaca** [comment]
No whiteboard in instructions.  All but one test are good.  Any help with left and right node test?

File: java/datastructures/lib/src/test/java/datastructures/linkedlist/BinaryTreeTests.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  @Test void testAddRoot(){
+    BinarySearchTree<Integer> newTree = new BinarySearchTree<>();
+    newTree.add(1);
+    assertNotNull(newTree.root);
+  }
+
+  @Test void testAddLeftAndRightNode(){
+    BinarySearchTree<Integer> newTree = new BinarySearchTree<>();
+    newTree.add(1);
```
Comment (lucy-gelderloos) [inline java/datastructures/lib/src/test/java/datastructures/linkedlist/BinaryTreeTests.java:25]: Try going through this step by step. If 1 is the first value added, the value of the root node is going to be 1. If all the values you add after that are larger than 1, they'll all get sorted to the right and `newTree.root.leftNode` will always be null.
